scripts/ci/e2e-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
@gavin-stackrox I didn't know where else to put this because I had to put `e2e_tests.py` in `scripts/ci/jobs`. Perhaps I should put this under something like `scripts/ci/jobs/e2etests`? I think it's just a little awkward because this is essntially the same script as `scripts/ci/jobs/unit-tests.sh`

Reply:
I put it in `scripts/ci/jobs/e2etests` but let me know if you'd prefer something else

Reply:
@RTann I think it is fine there.

This python layer makes it a bit awkward. On the stackrox side the intent was not to repeat the shared behavior among the e2e tests but given that scanner has 1 e2e test, going `dispatch.sh -> e2e_tests.py -> ci_tests.py -> E2eTest() -> e2e-tests.sh` is confusing.

Reply:
I'm planning on 1-2 follow up PR(s) to add support for `slim-e2e-tests` and `scale-tests` which will require the same infrastructure as the `e2e-tests`. They will probably also need very similar `.sh` files as well